tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -1,5 +1,3 @@
-# flake8: noqa
-data = """---
 namespace.tendrl.ceph_integration:
```

Comment:
This should be just "namespace.ceph", 

Final namespaces are eg: namespace.tendrl (commons), namespace.node_agent (node_agent), namespace.provisioning, namespace.integrations.ceph (in the node_agent and these will hold the ceph, gluster specific import/create flows/atoms/objects, which will be moved from commons), namespace.performance_monitoring, namespace.node_monitoring, namespace.alerting, namespace.ceph (ceph bridge), namespace.gluster (gluster bridge)

cc: @nthomas-redhat @anivargi @anmolbabu @GowthamShanmugam @nnDarshan @shtripat @anmolsachan  fyi